server/index.js

```diff
@@ -22,17 +25,28 @@ app.get('/tetraflix/recommendations/:user', (req, res) => {
 });
 
 app.get('/tetraflix/genre/:genre', (req, res) => {
-  // possible genres: Action, Animation, Comedy, Documentary, Drama,
-  // Family, Fantasy, Foreign, Horror, Musical, Mystery,
-  // Romance, Science Fiction, Thriller, Western
-
   // genre recs are not user-specific
   const genre = req.params.genre;
 
   // look up recs by genre
   // something like: queryResult = findOne({ where: { genre: genre } });
-  
+
   // very simplified result; actual will contain 20 movie objects
   const queryResult = '"{"genre":[{"id":34532,"title":"Spider Man","profile":{"action":80,"comedy":20},"progress":0},{"id":567490,"title":"Star Wars","profile":{"action":70,"fantasy":30},"progress":0}]}"';
   res.send(queryResult);
 });
+
+app.get('/tetraflix/dummyData/movies', (req, res) => {
+  pgDummyData();
+  res.send('adding movies...');
+});
+
+app.get('/tetraflix/dummyData/userMovies', (req, res) => {
+  mgDummyData();
+  res.send('adding user...');
+});
+
+// app.get('/tetraflix/testing', (req, res) => {
```

Comment:
Do you still need this?

Reply:
I was using that endpoint to see if I could write a test for the server.  I got it to work, but I ended up querying the databases directly so I did not use this.  I want to keep it here as a model for when I write server tests.  Then I will get rid of it.